src/style.css

```diff
@@ -1,3 +1,53 @@
+html {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+
+  --my-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+header {
+  height: 60px;
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.imgContainer {
+  margin: 10px 0 10px 50px;
+}
+
+ul {
+  display: flex;
+  justify-content: space-evenly;
+  margin-right: 50px;
+}
+
+li {
+  list-style: none;
+  height: 15px;
+  margin-left: 13px;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+a.is-active {
+  text-decoration: underline 4px;
+  text-underline-offset: 1.2rem;
```

Comment:
There is requirement for that underline in mockup: -Use the `::after` and position it relative to a link with is-active class